crates/bevy_scene/src/lib.rs

```diff
@@ -40,9 +41,7 @@ impl Plugin for ScenePlugin {
             .add_asset::<Scene>()
             .init_asset_loader::<SceneLoader>()
             .init_resource::<SceneSpawner>()
-            .add_systems(Update, scene_spawner_system)
-            // Systems `*_bundle_spawner` must run before `scene_spawner_system`
-            .add_systems(PreUpdate, scene_spawner);
+            .add_systems(PreUpdate, (scene_spawner, scene_spawner_system).chain());
```

Comment:
```suggestion
            .add_systems(PostUpdate, (scene_spawner, scene_spawner_system).chain().before(propagate_transform));
```

Reply:
We discussed it in Discord and decided that it's better to put it in `PostUpdate` to have fully updated transform.
@lewiszlw I really sorry you had to change it so many times :smile:
But this one you can't apply from the browser, you need to import `propagate_transform`.

Reply:
No worries. Should be `.before(TransformSystem::TransformPropagate)`?

Reply:
Yes, looks like this is what mostly used in the codebase.

Reply:
Before you make any change, take a look at what cart suggesting, I think it makes sense: https://github.com/bevyengine/bevy/pull/9260#discussion_r1286374503
Because both solution (`PreUpdate` and `PostUpdate`) have downsides, this is why we have such hard time deciding it.